Editor/Window/RegisterComputeInput.cs

```diff
@@ -38,10 +43,11 @@ public RegisterComputeInput(VisualElement container, StateManager stateManager,
             _ipInputs = container.Query<TextField>("AnywherePageComputeIPAddressInput").ToList();
             _computeStatus = container.Q("AnywherePageComputeStatus");
             _registerButton = container.Q<Button>("AnywherePageComputeRegisterButton");
-            _registerNewButton = container.Q<Button>("AnywherePageComputeRegisterNewButton");
-            _cancelButton = container.Q<Button>("AnywherePageComputeCancelButton");
+            _replaceComputeButton = container.Q<Button>("AnywherePageComputeRegisterNewButton");
```

Comment:
Should this variable name be `registerNewButton`? Doesn't match the element name it is currently querying

Reply:
Alternatively, if the element name has to be switched, that works too.